### PR TITLE
Update for docker checkpoint

### PR DESCRIPTION
**Pull request by yuexiao-wang**
**- What I did**
1. Add option in Usage
2. Modify the description by upper-case letter
3. Modify the subtitle in checkpoint-restore.md 

**- How I did it**

**- How to verify it**

**- Description for the changelog**

Signed-off-by: yuexiao-wang <wang.yuexiao@zte.com.cn>

### CODE REVIEW

File: experimental/checkpoint-restore.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ Alternatively, you can [build CRIU from source](http://criu.org/Installation).
 
 You need at least version 2.0 of CRIU to run checkpoint/restore in Docker.
 
-## Use cases for checkpoint & restore
+## Using cases for checkpoint & restore
```
Comment (AkihiroSuda) [inline experimental/checkpoint-restore.md]: "Use cases" is valid

**Review by AkihiroSuda — CHANGES_REQUESTED** [review]

File: experimental/checkpoint-restore.md (hunk 1 above)
Comment (yuexiao-wang) [inline experimental/checkpoint-restore.md]: Thanks, I will apply it to the PR

Comment (yuexiao-wang) [inline experimental/checkpoint-restore.md]: @AkihiroSuda PTAL
Updated

File: experimental/checkpoint-restore.md [hunk 2]
```diff
@@ -48,7 +48,8 @@ The options for checkpoint create:
 
     Create a checkpoint from a running container
 
-      --leave-running=false    leave the container running after checkpoint
+      --leave-running=false    Leave the container running after checkpoint
+	  --checkpoint-dir         Use a custom checkpoint storage directory
```
Comment (AkihiroSuda) [inline experimental/checkpoint-restore.md]: Indentation seems corrupted

Comment (yuexiao-wang) [inline experimental/checkpoint-restore.md]: @AkihiroSuda  I copy it  from the code,  but it works good  as fellows:  https://github.com/yuexiao-wang/docker/blob/update-checkpoint/experimental/checkpoint-restore.md   
What do you have suggestion?

Comment (yuexiao-wang) [inline experimental/checkpoint-restore.md]: Have done and thanks!

**Review by AkihiroSuda — APPROVED** [review]
LGTM

**Review by vdemeester — APPROVED** [review]
LGTM 🐮

**Comment by AkihiroSuda** [comment]
CI green, merging

**Pull request merged**